tests/ui/fn/fn-arg-count-mismatch-diagnostics.stderr

```diff
@@ -83,6 +83,32 @@ help: provide the arguments
 LL |     bar(1, 2, 3, /* i32 */, /* i32 */, /* i32 */);
    |                +++++++++++++++++++++++++++++++++
 
-error: aborting due to 5 previous errors
+error[E0061]: this function takes 6 arguments but 5 arguments were supplied
+  --> $DIR/fn-arg-count-mismatch-diagnostics.rs:58:5
+   |
+LL |     function_with_lots_of_arguments(
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+LL |         variable_name,
+LL |         variable_name,
+   |         ------------- argument #2 of type `char` is missing
+   |
+note: function defined here
+  --> $DIR/fn-arg-count-mismatch-diagnostics.rs:49:4
+   |
+LL | fn function_with_lots_of_arguments(a: i32, b: char, c: i32, d: i32, e: i32, f: i32) {}
+   |    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^         -------
+help: provide the argument
+   |
+LL ~     function_with_lots_of_arguments(
+LL +         variable_name,
+LL +         /* char */,
+LL +         variable_name,
+LL +         variable_name,
+LL +         variable_name,
+LL +         variable_name
```

Comment:
I don't understand why the suggestion shows the `variable_name` arguments as `+` (added). I tried to look around and tweak the code a bit, but found nothing :/